pom.xml

```diff
@@ -1282,6 +1371,9 @@
         <forgetme.tool.version>1.1.20</forgetme.tool.version>
 
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <!-- Feature versions -->
+        <is.analytics.feature.version>1.0.0-SNAPSHOT</is.analytics.feature.version>
```

Comment:
We have introduced a version property for all analytics solutions: [`analytics.solutions.version`](https://github.com/wso2/product-sp/blob/368ee6570df88ab7db55ac2a9a351325c648cbf9/pom.xml#L1107) 

Could you please remove  this property and use `analytics.solutions.version` instead?

Reply:
Done